Sming/third-party/.patches/esp-open-lwip.patch

```diff
@@ -144,3 +125,46 @@ index eaa8dd6..6a08192 100644
  #endif
  #endif
  
+diff --git a/include/lwip/ip_addr.h b/include/lwip/ip_addr.h
+index 1e46ee5..cfc10f8 100644
+--- a/include/lwip/ip_addr.h
++++ b/include/lwip/ip_addr.h
+@@ -210,7 +210,7 @@ u8_t ip4_addr_netmask_valid(u32_t netmask)ICACHE_FLASH_ATTR;
+ #define ip_addr_islinklocal(addr1) (((addr1)->addr & PP_HTONL(0xffff0000UL)) == PP_HTONL(0xa9fe0000UL))
+ 
+ #define ip_addr_debug_print(debug, ipaddr) \
+-  LWIP_DEBUGF(debug, ("%"U16_F".%"U16_F".%"U16_F".%"U16_F,             \
++  LWIP_DEBUGF(debug, ("%" U16_F ".%" U16_F ".%" U16_F ".%" U16_F,             \
+                       ipaddr != NULL ? ip4_addr1_16(ipaddr) : 0,       \
+                       ipaddr != NULL ? ip4_addr2_16(ipaddr) : 0,       \
+                       ipaddr != NULL ? ip4_addr3_16(ipaddr) : 0,       \
+diff --git a/include/lwip/mem.h b/include/lwip/mem.h
+index af6e360..6d8cabd 100644
+--- a/include/lwip/mem.h
++++ b/include/lwip/mem.h
+@@ -52,19 +52,19 @@ typedef size_t mem_size_t;
+  */
+ #ifndef MEMLEAK_DEBUG
+ #ifndef mem_free
+-#define mem_free vPortFree
++#define mem_free(s) vPortFree(s, "", 0)
```

Comment:
Why use empty string,0 when memleak is not defined? why not use simpler function version?